src/ripple/core/impl/Config.cpp

```diff
@@ -357,13 +360,62 @@ void Config::loadFromString (std::string const& fileContents)
 
     if (getSingleSection (secConfig, SECTION_VALIDATION_SEED, strTemp, j_))
     {
+        if (exists(SECTION_VALIDATOR_TOKEN))
+            Throw<std::runtime_error> (
+                "Cannot have both [" SECTION_VALIDATION_SEED "] "
+                "and [" SECTION_VALIDATOR_TOKEN "] config sections");
+
         auto const seed = parseBase58<Seed>(strTemp);
         if (!seed)
             Throw<std::runtime_error> (
                 "Invalid seed specified in [" SECTION_VALIDATION_SEED "]");
         VALIDATION_PRIV = generateSecretKey (KeyType::secp256k1, *seed);
         VALIDATION_PUB = derivePublicKey (KeyType::secp256k1, VALIDATION_PRIV);
     }
+    else if (exists(SECTION_VALIDATOR_TOKEN))
+    {
+        std::string tokenStr;
+        for (auto const& line : section (SECTION_VALIDATOR_TOKEN).lines())
+            tokenStr += beast::rfc2616::trim(line);
+
+        tokenStr = beast::detail::base64_decode(tokenStr);
```

Comment:
I think I'd probably feel okay with the config returning a reference to a `const ValidatorToken` object that did the parsing and kept the data.  It just feels funny for the config to be so intimate with the details of a non-human-readable field.  You could also try to convince me that what the config is doing now is okay, if it doesn't feel odd to you.